tests/jballerina-unit-test/src/test/resources/test-src/statements/transaction/transaction_block_negative.bal

```diff
@@ -11,32 +11,6 @@ function testCorrectTransactionBlock() returns (string) {
     return a;
 }
 
-function testTwoAbortedBlocks() returns (string) {
-    string a = "";
-    transaction with retries=2 {
-    } onretry {
-        a = a + " retry";
-    } aborted {
```

Comment:
With current grammer impl, multiple aborted/committed blocks are prohibited from the grammer level.